docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -119,27 +119,29 @@ Once your cluster administrator has :ref:`setup a Kubernetes cluster <kubernetes
     $ kubectl config set-context --current --namespace=mynamespace
 
 
-.. tip::
 
-  You can list the GPUs available on your cluster with :code:`sky show-gpus --cloud kubernetes`.
+Listing Available GPUs
+----------------------
 
-  .. code-block:: console
+You can list the GPUs available on your cluster with :code:`sky show-gpus --cloud kubernetes`.
```

Comment:
Can we also suggest + have a snippet below for `sky show-gpus`? It'll show k8s info if enabled. Reason is it's shorter to type?

Reply:
Updated to use `sky show-gpus`.